scripts/saveImageDay.sh

```diff
@@ -69,33 +71,14 @@ fi
 # If upload is true, optionally create a smaller version of the image; either way, upload it
 if [ "$UPLOAD_IMG" = true ] ; then
 	if [[ "$RESIZE_UPLOADS" == "true" ]]; then
-		echo -e "$ME: Resizing '$IMAGE_TO_USE' for uploading"
-		echo -e "$ME: Resizing '$IMAGE_TO_USE' for uploading\n" >> log.txt
-
 		# Create a smaller version for upload
 		convert "$IMAGE_TO_USE" -resize "$RESIZE_UPLOADS_SIZE" -gravity East -chop 2x0 "$IMAGE_TO_USE"
 		RET=$?
 		if [ $RET -ne 0 ] ; then
-			echo "*** $ME: ERROR: RESIZE_UPLOADS failed with RET=$RET"
-			exit 1
+			echo -e "${YELLOW}*** ${ME}: WARNING: RESIZE_UPLOADS failed with RET=${RET}; continuing with larger image.${NC}"
 		fi
 	fi
 
-	TS=$(ls -l --time-style='+%H:%M:%S' "$IMAGE_TO_USE" | awk '{print $6}')
-	echo -e "$ME: Uploading '$IMAGE_TO_USE' with timestamp: $TS\n"
-	echo -e "$ME: Uploading '$IMAGE_TO_USE' with timestamp: $TS" >> log.txt
-	if [[ $PROTOCOL == "S3" ]] ; then
-                $AWS_CLI_DIR/aws s3 cp "$IMAGE_TO_USE" s3://$S3_BUCKET$IMGDIR --acl $S3_ACL &
-        elif [[ $PROTOCOL == "local" ]] ; then
-		cp "$IMAGE_TO_USE" "$IMGDIR" &
-	else
-		# Put to a temp name, then move the temp name to the final name.
-		# This is because slow uplinks can cause multiple lftp requests to be running at once,
-		# and only one lftp can upload the file at once, otherwise get this error:
-		# put: Access failed: 550 The process cannot access the file because it is being used by another process.  (image.jpg)
-		# Slow uploads also cause a problem with web pages that try to read the file as it's being uploaded.
-		# "si" = "save image" - use a unique temporary name
-		TEMP_NAME="si-$RANDOM"
-		lftp "$PROTOCOL://$USER:$PASSWORD@$HOST:$IMGDIR" -e "set net:max-retries 2; set net:timeout 20; put "$IMAGE_TO_USE" -o $TEMP_NAME; rm -f "$IMAGE_TO_USE"; mv $TEMP_NAME "$IMAGE_TO_USE"; bye" &
-        fi
+	"${ALLSKY_SCRIPTS}/upload.sh" "${IMAGE_TO_USE}" "${IMGDIR}" "${IMAGE_TO_USE}" "SI"
```

Comment:
"SI"? "NO!" ;)

But seriously, what is "SI"? Could it be named something more descriptive?

Reply:
Stands for "Save Image".  It's a unique ID that has a random number appended to it by upload.sh so lftp doesn't overwrite another lftp's file.  The user should never see the string.